### PR TITLE
Update revision history for 0.570

### PR DESCRIPTION
**Pull request by msullivan**

### CODE REVIEW

**Comment by gvanrossum** [comment]
LGTM. That's the only doc change (apart from nits)?

File: docs/source/revision_history.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@ Revision history
 
 List of major changes:
 
+- March 2017
```
Comment (JelleZijlstra) [inline docs/source/revision_history.rst]: I think it's 2018 now :)

**Comment by msullivan** [comment]
The only doc change I had planned at least. Attrs docs and making strict optional and incremental non-experimental went in before the branch cut.

**Comment by gvanrossum** [comment]
Go ahead, merge and then CP into the release branch. (Thanks Jelle for catching the mistaken year!)

**Pull request merged**